src/languages/es-ES.ts

```diff
@@ -2519,12 +2547,12 @@ export default class extends Language {
 		COMMAND_8BALL_NOT_QUESTION: 'Eso no parece ser una pregunta...',
 		COMMAND_8BALL_QUESTIONS: {
 			QUESTION: '?',
-			WHEN: /^¿?cu[áa]ndo/i,
-			WHAT: /^¿?qu[ée]/i,
-			HOW_MUCH: /^¿?cu[áa]nto/i,
-			HOW_MANY: /^¿?cu[áa]nto/i,
-			WHY: /^¿?por qu[ée]/i,
-			WHO: /^¿?qui[ée]n/i
+			WHEN: 'cuando',
+			WHAT: 'qué',
+			HOW_MUCH: 'cuanto',
+			HOW_MANY: 'cuantos',
+			WHY: 'por qué',
+			WHO: 'quien'
```

Comment:
They now match the same keys for English where they were literal strings rather than Regex expressions as well. Either one of them had to have been wrong.

Reply:
That object supports either strings or RegExp, please revert that change